### PR TITLE
Simplified uptime check

### PR DESCRIPTION
**Pull request by richir-sunet**
A simplified uptime check script, with slightly different output. Old one also had a bug and was very hard to read.

### CODE REVIEW

File: manifests/nagios.pp [hunk 1]
```diff
@@ -90,7 +90,7 @@
     command_line => "/usr/lib/nagios/plugins/check_procs -k -w ${_procw} -c ${_procc}"
   }
   sunet::nagios::nrpe_command {'check_uptime':
-    command_line => '/usr/lib/nagios/plugins/check_uptime.pl -f'
+    command_line => '/usr/lib/nagios/plugins/check_uptime.py'
```
Comment (mickenordin) [inline manifests/nagios.pp:93]: and here it would be:
```
command_line => "/usr/lib/nagios/plugins/check_uptime.py -w ${uptimew} -c ${uptimec}"
```

**Review by mickenordin — COMMENTED** [review]
A suggestion about overriding the default values

File: manifests/nagios/nrpe_check_uptime.pp
[the remark this answers is not in the document]
Comment (richir-sunet) [reply manifests/nagios/nrpe_check_uptime.pp:1]: Sounds like a good idea.

**Review by mickenordin — CHANGES_REQUESTED** [review]
Two bug on line 95 in manifests/nagios.pp

You need double quotes around the command and a dollar sign is missing on {uptime}

**Review by mickenordin — APPROVED** [review]
Looking good!

**Pull request merged**